src/sys/sendfile.rs

```diff
@@ -125,6 +121,44 @@ cfg_if! {
                 )
             }
         }
+    } else if #[cfg(solarish)] {
+        use std::os::unix::io::BorrowedFd;
+
+        #[derive(Debug, Copy, Clone)]
+        /// Mapping of the raw C sendfilevec_t struct
+        pub struct SendfileVec<'a> {
+            raw: libc::sendfilevec_t,
+            phantom: PhantomData<BorrowedFd<'a>>
+        }
+
+        const SFV_FD_SELF: i32 = -2;
```

Comment:
Constants like this should be defined in the libc crate, not in Nix.

Reply:
Yes I know I just did a PR for it tough :)